src/components/settings/ReportsTab.vue

```diff
@@ -28,12 +28,64 @@
             ></v-switch>
           </div>
         </div>
-      </v-col>
-    </v-row>
+        <div class="d-flex align-start mt-4">
+          <div class="flex-grow-1">
+            <p
+              class="subtitle-1 mb-2"
+              :style="{ color: currentTheme.secondary }"
+            >
+              {{ $tc("caption.add_org_logo_in_pdf", 1) }}
+            </p>
+            <p class="caption mb-0">
+              {{ $t("message.add_organization_logo") }}.
+            </p>
+          </div>
+          <div class="flex-grow-0">
+            <v-switch
+              v-model="config.reportLogo"
+              inset
+              hide-details
+              dense
+              class="mt-0 pt-0"
+              @change="handleConfig"
+            ></v-switch>
+          </div>
+        </div>
+      </div>
+      <v-card
+        class="mx-2 my-2 d-flex flex-column align-center selected"
+        max-width="200"
+        max-height="300"
+        theme="dark"
+        title="Card title"
+        :disabled="!config.reportLogo"
+        link
+        hover
+        @click.stop="openImage"
+      >
+        <v-img
+          :aspect-ratio="1"
+          :src="require('../../assets/icon/plus.png')"
+          class="bg-white"
+          width="80px"
+          cover
+        ></v-img>
+        <v-divider class="" width="200px"></v-divider>
+        <v-img
+          class="bg-white"
+          :src="require('../../assets/new_logo.png')"
```

Comment:
This shouldn’t be hard coded.

the src should be a `file://` link if `config.reportLogo` is set.

Like [here](https://github.com/yatt-ai/yattie/blob/3a1f1e12cd96ce3be7d3482caa06fb62391c652f/src/views/PrintView.vue#L113)